src/app/footer-nav/cookies.component.html

```diff
@@ -12,7 +12,7 @@ <h1 class="heading-large">Cookies</h1>
           when you visit a website. Most websites need cookies to work properly.</span>
       </div>
 
-      <h3 class="heading-medium">How cookies are used in this service</h3>
+      <h3 class="heading-h3">How cookies are used in this service</h3>
```

Comment:
Can't we create a common class, say .cookies-header-size, set it up with a heading-h3, and use that one here? that way it'll be simpler changing it in the future if needed

Reply:
Since I am using standard h3 header across the CCD including cookies, I am using same class here and in the rest of the places as well